tests/lua/type_mismatch.lua

```diff
@@ -153,6 +161,10 @@ end
 
 -- Put it to function to limit the lifetime of objects
 generate_tests()
+-- Remove garbage after generation
+effil.cache.clear()
```

Comment:
Move this to taer_down.

Reply:
There is special comment which explain why it's placed here: "Remove garbage after generation".
Here we clean up objects that was created on generation step, so, their lifetime wider than test case and tear down will not delete them.